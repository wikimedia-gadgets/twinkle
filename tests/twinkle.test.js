'use strict';

describe('modules/twinkle', () => {
	describe('Twinkle.defaultConfig global variables', () => {
		test('Should match snapshot', () => {
			expect(Twinkle.defaultConfig.welcomeUserOnSpeedyDeletionNotification).toMatchSnapshot();
			expect(Twinkle.defaultConfig.notifyUserOnSpeedyDeletionNomination).toMatchSnapshot();
			expect(Twinkle.defaultConfig.warnUserOnSpeedyDelete).toMatchSnapshot();
		});
	});
});
