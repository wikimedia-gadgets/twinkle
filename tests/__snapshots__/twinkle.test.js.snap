// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modules/twinkle Twinkle.defaultConfig global variables Should match snapshot 1`] = `
[
  "db",
  "g1",
  "g2",
  "g3",
  "g4",
  "g6",
  "g10",
  "g11",
  "g12",
  "g13",
  "g14",
  "a1",
  "a2",
  "a3",
  "a7",
  "a9",
  "a10",
  "a11",
  "c1",
  "f1",
  "f2",
  "f3",
  "f7",
  "f9",
  "r3",
  "u5",
]
`;

exports[`modules/twinkle Twinkle.defaultConfig global variables Should match snapshot 2`] = `
[
  "db",
  "g1",
  "g2",
  "g3",
  "g4",
  "g6",
  "g10",
  "g11",
  "g12",
  "g13",
  "g14",
  "a1",
  "a2",
  "a3",
  "a7",
  "a9",
  "a10",
  "a11",
  "c1",
  "f1",
  "f2",
  "f3",
  "f7",
  "f9",
  "r3",
  "u5",
]
`;

exports[`modules/twinkle Twinkle.defaultConfig global variables Should match snapshot 3`] = `
[
  "db",
  "g1",
  "g2",
  "g3",
  "g4",
  "g6",
  "g10",
  "g11",
  "g12",
  "g13",
  "g14",
  "a1",
  "a2",
  "a3",
  "a7",
  "a9",
  "a10",
  "a11",
  "c1",
  "f1",
  "f2",
  "f3",
  "f7",
  "f9",
  "r3",
  "u5",
]
`;
